1-js/09-classes/02-class-inheritance/article.md

```diff
@@ -32,7 +32,7 @@ Así es como podemos representar gráficamente el objeto `animal` y la clase `An
 
 ![](rabbit-animal-independent-animal.svg)
 
-...Y nos gustaría crear otra clase `Rabbit`.
+...Y nos gustaría crear otra clase `Rabbit` (Conejo).
```

Comment:
Creo que es innecesaria la traducción de "Rabbit"

Reply:
por?
Ok, lo saqué.

Dudé cuando lo agregué, luego pensé en alguien que no sabe NADA de ingles, que no vio nunca RABBIT, un genio de 6 años por ej,

Ademas de las lineas agregadas veras que revise cosas que me parecian mal, como  "this" traducido.
en lass palabras ingles, me fijé en las "primeras ocurrencias", era la unica que no se menciono
quizass estoy lo que le dicen "overthinking"